var/spack/repos/builtin/packages/pkg-config/package.py

```diff
@@ -24,23 +24,29 @@
 ##############################################################################
 from spack import *
 
+
 class PkgConfig(Package):
-    """pkg-config is a helper tool used when compiling applications and libraries"""
+    """pkg-config is a helper tool used when compiling applications
+    and libraries"""
+
     homepage = "http://www.freedesktop.org/wiki/Software/pkg-config/"
     url      = "http://pkgconfig.freedesktop.org/releases/pkg-config-0.28.tar.gz"
 
-    version('0.28', 'aa3c86e67551adc3ac865160e34a2a0d')
+    version('0.29.1', 'f739a28cae4e0ca291f82d1d41ef107d')
+    version('0.28',   'aa3c86e67551adc3ac865160e34a2a0d')
 
     parallel = False
 
+    # The following patch is needed for gcc-6.1
+    patch('g_date_strftime.patch')
```

Comment:
This patch is identical to the one provided for glibc. Since there is a circular dependency between glibc and pkg-config, we were using the internal glibc provided by pkg-config, which wasn't patched.